app/assets/javascripts/surveys.js

```diff
@@ -189,6 +193,65 @@
       }
     },
 
+    parsedUrlSurveyTemplate: function() {
+      return {
+        title: "Tell us what you think of GOV.UK",
+        surveyCta: "Take the 3 minute survey",
+        surveyCtaPostscript: "This will open a short survey on another website",
+        render: function(survey) {
+          var templateArgs = (survey.templateArguments || {}),
+              surveyUrl = userSurveys.addCurrentPathtoURL(survey);
+
+          // Smart survey can record the URL of the survey link if passed through as a query param
+          if ((/smartsurvey/.test(surveyUrl)) && (surveyUrl.indexOf('?c=') === -1)) {
```

Comment:
Doesn't the `{{currentPath}}` template param stuff handle this?  I think we can drop this.